api/types/resetpasswordtoken_test.go

```diff
@@ -75,15 +74,11 @@ func (s *ResetPasswordTokenSuite) TestUnmarshal(c *check.C) {
 		comment := check.Commentf("test case %q", tc.description)
 		out, err := marshaler.Unmarshal([]byte(tc.input))
 		c.Assert(err, check.IsNil, comment)
-		if !reflect.DeepEqual(tc.expected, out) {
-			c.Fatalf("expected %v, but got %v", tc.expected, out)
-		}
+		c.Assert(out, check.DeepEquals, tc.expected, check.Commentf(tc.description))
```

Comment:
nit: I did not notice it before, but there's already a `comment` (value of type check.Commentf) which you can use here:
```go
c.Assert(out, check.DeepEquals, tc.expected, comment)
```